### ISSUE TITLE
[A11yInspect] Fail: 4.1.2 A in /home/runner/work/test-app/test-app/src/App.tsx

### ISSUE THREAD

**Issue by sitaram-r** [sitaram-r/test-app#22]
## ❌ Accessibility Fail

**Rule:** 4.1.2 A
**File:** `/home/runner/work/test-app/test-app/src/App.tsx`
**Line:** 1, **Column:** 1

### Description
Missing accessible name for checkbox (Critical)

### Context
```html
<input type="checkbox">
```

### Selector
`html > body > div > audio > input:nth-of-type(2)`

---
*Generated by [A11yInspect](https://github.com/barrierbreak/vscode-scally)*
